### ISSUE TITLE
Post the created files

### ISSUE THREAD

**Issue by mrcarlosrendon** [ssoper/zip-code-boundaries#1]
Is there any reason to not just post the output files as well?

**Comment by JHKennedy4** [comment]
Yeah, that would be really useful, and save everyone a few minutes. Thanks for the script.

**Comment by JHKennedy4** [comment]
@mrcarlosrendon I added the data in my fork. Put up a pull request. If you want it it's there.

**Comment by KrunchMuffin** [comment]
is there a 2010 version yet?
